api/session_test.go

```diff
@@ -21,6 +21,13 @@ var _ = Describe("Session", func() {
 		session = &Session{bus}
 	})
 
+	Describe("#New", func() {
```

Comment:
I am looking at the ways on how to extract SessionID using the approach described above but I cannot use .Send method for that due to below limitation with unmarshal code. The response, coming from WebDriver, has sessionId defined as part of the body (not part of the value object). 

As an option I can make .Send method to return an error which would intentionally trigger an error contain the whole response body section :-) to parse. Let me know what you think.

**bodyValue := struct{ Value interface{} }{result}**
if err := json.Unmarshal(responseBody, &bodyValue); err != nil {
	return fmt.Errorf("unexpected response: %s", responseBody)
}


![image](https://cloud.githubusercontent.com/assets/7816406/20580193/f5acc096-b19e-11e6-85df-db3aa9a58199.png)